src/LightGraphs.jl

```diff
@@ -20,7 +20,7 @@ import SparseArrays: blockdiag, sparse
 import Base: adjoint, write, ==, <, *, ≈, convert, isless, issubset, union, intersect,
             reverse, reverse!, isassigned, getindex, setindex!, show,
             print, copy, in, sum, size, eltype, length, ndims, transpose,
-            join, iterate, eltype, get, Pair, Tuple, zero
+            join, iterate, eltype, get, Pair, Tuple, zero, push!, popfirst!, pushfirst!
```

Comment:
It shouldn't be necessary to import  `push! , `popfirst! ` and `pushfirst` - the reason that we import the other functions from `Base` here is that we overload these functions for our types and for that you have to explicitly import them.

Reply:
Ok, cool! I'm scraping them!